pkgs/applications/editors/trse/default.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+, libGL
+, lua
+, qmake
+, qtbase
+, wrapQtAppsHook
+}:
+
+stdenv.mkDerivation {
+  name = "trse";
+  version = "0.13.337";
+
+  src = fetchFromGitHub {
+    owner = "leuat";
+    repo = "TRSE";
+    rev = "a06ab438db51e31eedf1c9329316672783e7b9e8";
+    sha256 = "1hmihjrcvlwvr39737rbvl6nyib1mxzxbbjxfqxpmhw6n14qgi4p";
+  };
+
+  nativeBuildInputs = [
+    qmake
+    wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    libGL
+    lua
+    qtbase
+  ];
+
+  qmakeFlags = [
+    "TRSE.pro"
+  ];
+
+  postInstall = ''
```

Comment:
Are project templates and the tutorials meant to be used by the user? 

Maybe $out/share/doc is more applicable

Reply:
> Are project templates and the tutorials meant to be used by the user?

They are, and they need to be in that exact location, it seems: https://github.com/leuat/TRSE/blob/master/README.md

Reply:
The tutorials are hardwired to the front page of the IDE, so I think it's right to include them. That means they can't be changed (or built) from within the program.